### DISCUSSION TITLE
Creating a Release shows the right date of the release files, but the contents are older.

### DISCUSSION THREAD

**Discussion by GuruSR** [community/community#116788] [Repositories] [answered]
### Select Topic Area

Bug

### Body

I have run into this a few times, sadly, I've just deleted all the Releases.  I did get one repository to come out of this behavior.

On the web, this is what happens (all on the master, no tag):
  I make a new release (tag created on creation of release), this works.
  The next release was then created the same way down the road (in this case January 12th).

That second release was supposed to be the January 12th changes, but what is inside the source zip file in the release is the previous release changes.  I deleted the release and tried again, unchanged.

I then removed all the releases and started a brand new set, starting at the January 12th release.  Sadly, it is still showing the right date on the file, but the contents are now from the removed previous release changes.

How can I get around this, so the releases can be used again?

**Comment by giovannivicentin** [comment]
Hello GuruSR,

It sounds like you're experiencing an issue where the source code ZIP file attached to your GitHub releases is not reflecting the correct state of the repository at the point of the tag. Here are some steps you can take to troubleshoot and resolve this issue:

1. **Confirm Tag Points**: Ensure that the tags are pointing to the correct commits. You can do this by checking out the tag locally and comparing the contents to what you expect.

2. **Manual Release**: Try creating a release manually without using any actions or automation tools. This can help determine if the problem is with the automation or with GitHub itself.

3. **Tag Creation**: When you create a release, GitHub automatically creates a tag if it doesn't exist. Make sure that if you're tagging manually, you're doing so at the correct commit. Also, check to see if the automatic tagging by GitHub is functioning correctly.

4. **GitHub Actions**: If you're using GitHub Actions to automate release creations, carefully review your workflow files. Ensure that the actions are checking out the correct commit or branch and that any build or packaging scripts are running against the right code.

5. **Cache Issue**: There might be a caching issue with GitHub. Sometimes the ZIP files can be cached improperly, showing old code. Try downloading the ZIP file in an incognito window or clearing your browser's cache before downloading.

6. **GitHub Support**: Since you've tried deleting and recreating releases without success, this could be a more persistent issue on the GitHub platform. Contact GitHub Support with detailed information about your repository, the releases in question, and the steps you've taken so far.

7. **Documentation and Release Notes**: Document this behavior and your attempts to resolve it. This documentation will be useful when reaching out to GitHub Support and may help other users experiencing the same issue.

8. **Verify with API**: Use the GitHub API to fetch the release information and check if the API's data is consistent with what you're seeing on the web interface. If there's a discrepancy, this might indicate a synchronization issue on GitHub's side.

9. **Alternative Release Method**: As a temporary workaround, you could manually attach ZIP files to the release. Generate the ZIP file locally from the correct commit and upload it as an asset to the release.

Unfortunately, this seems to be a backend issue that may require GitHub's intervention. However, the steps above should help in identifying or working around the problem until it can be properly addressed by GitHub.

**Reply by GuruSR** [reply]
Yes.

1. Only tag currently in the repository is the version I tried to re-create, as I removed all of the releases and tags associated with them.   (No releases or tags present.)
2. I am only doing manual releases (on the website).
3. Tag creation was done at the manual release being created automatically at time of commit of the release.
4. Not using this.
5. Just tried it.  The files inside are dated correctly, the data inside the files aren't.  The version is 1.3 for the release, there are version 1.2 file data in the files, but dated that of the 1.3 version release.  I just tried it on incognito download and no change.
6. I'll contact them, but I got the loop back to here, any quick way of doing this?  I've run into this issue before on another repository.
7. I'll do that.
8. I'll try Github Desktop, see if it can catch the proper information.
9. I'll try making a manual upload of the release.

I was able to solve this one as well, seems when I did a move of the files into another directory, it looks like the old version wound up inside.  I deleted the old versions and re-moved the proper ones inside the right folder again.  Only 1 copy now.

**Comment by giovannivicentin** [comment, marked as answer]
Regarding contacting GitHub Support and navigating through what might feel like a loop, here's a more direct approach you might find useful:

GitHub Support Portal: [GitHub Support Portal](https://support.github.com/) directly. From there, you can choose "Contact Support" or "Open a ticket" to get specific assistance. Be prepared to provide detailed information about your issue, including the steps you've already taken and the outcomes of those actions.

Direct Email: If the portal isn't yielding results or feels too circular, you could try contacting GitHub support directly via email at support@github.com. This route might be less structured, but it's another channel to reach out for help.

Twitter: Surprisingly, reaching out to companies on social media can sometimes expedite responses. Tweeting at or messaging [GitHub on Twitter](https://twitter.com/github) might get you the attention needed to address your issue more quickly.

**Reply by GuruSR** [reply]
Thanks, if I run into this issue again on another repository, I'll definitely be going those routes.
